enhancements/installer/ibm-cloud-for-power-vs-ipi.md

```diff
@@ -0,0 +1,231 @@
+---
+title: ibm-cloud-ipi
```

Comment:
should the title be changed to `powervs-ipi` to avoid ambiguity ?

Reply:
or `ibm-cloud-powervs-ipi`

Reply:
I went with ibm-cloud-for-power-vs-ipi in the latest revision, since it mirrors the updated title of the doc & PR.